pkg/handlers/login.go

```diff
@@ -110,7 +110,29 @@ func (h *Handler) Login(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	loginResponse.Token = fmt.Sprintf("Bearer %s", signedJWT)
+	responseToken := fmt.Sprintf("Bearer %s", signedJWT)
+
+	expire := time.Now().Add(24 * time.Hour)
```

Comment:
Looks like the token [expires after 12 hours](https://github.com/replicatedhq/kots/blob/main/pkg/handlers/login.go#L44), so the cookie expiration should probably align with that unless there is a reason not to.

Reply:
I updated the cookie to use the same `SessionTimeout` as the token.